pex/installer.py

```diff
@@ -32,12 +32,12 @@ def function_wrapper(self, *args, **kw):
 
 
 class InstallerBase(object):
-  SETUP_BOOTSTRAP_HEADER = "import sys"
+  SETUP_BOOTSTRAP_HEADER = "import sys; from codecs import open"
```

Comment:
any reason to not just use `io.open` here instead? it should be in python 2.6 and up and also supports an `encoding=` kwarg - and is the default `open` in python3.

I think you could also avoid the semicolon by using the multiple import syntax at the module level:

```
...
SETUP_BOOTSTRAP_HEADER = "import io, sys"
...
exec(compile(io.open(..., encoding='utf-8')...
```